src/app/_components/percentageCircle/percentageCircle.tsx

```diff
@@ -0,0 +1,70 @@
+import React from "react";
```

Comment:
Why do we need this component created? We have a mui circular progress bar with label for the same purpose

Reply:
done, although can't figure out colors 
mui created a customisation to show labels, no inbuilt component available with labels

Reply:
color?: OverridableStringUnion<
    'primary' | 'secondary' | 'error' | 'info' | 'success' | 'warning' | 'inherit',
    CircularProgressPropsColorOverrides
  >;

Reply:
you can use something like this sx={{ color: "red !important" }}